q2_sample_classifier/assets/index.html

```diff
@@ -70,7 +70,22 @@
       <a href="feature_importance.tsv">
         <p>Download as tsv</p>
       </a>
-      {{ importances }}
+    </div>
+    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+      <div class="panel panel-default">
+        <div class="panel-heading" role="tab" id="accordion-heading">
+          <h4 class="panel-title">
+            <a role="button" data-toggle="collapse" data-parent="#accordion" href="#table" aria-expanded="true" aria-controls="table">
+              Collapse
```

Comment:
Can you rename to 'Click to Collapse/Expand'? Users might not be familiar with this type of functionality so explicitly pointing out the behavior might be useful.